exercises/G-numbers/exercise.js

```diff
@@ -1 +1,14 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+let numberOfStudents = 15;
+let numberOfMentors = 8;
+let totalStudentsAndMentors = 0;
+let message = "";
+
+// Calculate total number of students
+totalStudentsAndMentors = numberOfStudents + numberOfMentors;
+
+// Concatenate string
+message = `Number of students: ${numberOfStudents}
+Number of mentors: ${numberOfMentors}
+Total number of students and mentors: ${totalStudentsAndMentors}`;
+console.log(message);
```

Comment:
Excellent. Again, you could do `let message = ` on line 11 and define the variable at the same time as you set its value. This can sometimes be safer, as it means you'll get an error message saying the variable is `undefined` if you try to use it before it has been set correctly. This is helpful as then you'll know there's something wrong.